client/snap_op.go

```diff
@@ -54,18 +61,18 @@ func (opts *SnapOptions) writeModeFields(mw *multipart.Writer) error {
 		{"dangerous", opts.Dangerous},
 	}
 	for _, o := range fields {
-		if !o.b {
-			continue
-		}
-		err := mw.WriteField(o.f, "true")
-		if err != nil {
+		if err := writeFieldBool(mw, o.f, o.b); err != nil {
 			return err
 		}
 	}
 
 	return nil
 }
 
+func (opts *SnapOptions) writeOptionFields(mw *multipart.Writer) error {
```

Comment:
You could rename `writeModeFields` to this, and just include Unaliased in that loop, right?

Reply:
Yes, although the code in API seems to split handling of mode flags from other options. I tried (maybe unnecessarily) to keep the same split here. I'm ok with rolling this into a single func if you feel strongly about it.

Reply:
I feel strongly about many things, but this is not one of them.